spec/public-api.yaml

```diff
@@ -22,6 +22,41 @@ paths:
               schema:
                 $ref: "#/components/schemas/User"
           description: ""
+  /v1/actions/detector/{id}/rules:
+    # TODO: leaving the GET endpoint out, to be consistent with detectors we probably want to paginate, but we'd need a backend change
+    post:
+      operationId: Create rule
+      description: Create a new rule for a detector.
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: string
+          description: Choose a detector by its ID.
+          required: true
+      tags:
+        - rules
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RuleCreationInput"
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: "#/components/schemas/RuleCreationInput"
+          multipart/form-data:
+            schema:
+              $ref: "#/components/schemas/RuleCreationInput"
```

Comment:
I think we only support `application/json`.

Reply:
This makes things easier